### DISCUSSION TITLE
identifiedAt attribute often not present

### DISCUSSION THREAD

**Discussion by ammerzon** [secureCodeBox/secureCodeBox#1100] [Q&A] [answered]
The [finding documentation](https://www.securecodebox.io/docs/api/finding) states that the `identified_at` attribute will often not be present. This information would be quite useful, and I wonder why it is not present more?

I assume this depends on the scanner, and there is no easy way to get to that information. Wouldn't it be helpful to get an attribute `started_at` for the scan so you have a duration with the already existing `parsed_at` attribute?

**Comment by J12934** [comment, marked as answer]
Yup totally agree that `identified_at` is usually the more interesting, but as you said we can only really set it if the scanner / the scanners result format includes it.

The scan currently doesn't really have a `started_at` timestamp, but all kubernetes objects have a `creationTimestamp` set. (`metadata.creationTimestamp`). This isn't necessarily the timestamp of the actual start of the scan as it might have been in a pending state if the cluster was currently unable to execute it, but it is usually relativly close.

**Reply by ammerzon** [reply]
Would it make sense to add this `creationTimestamp` to the scan even if it is a rough estimate of when the scan started?

**Reply by J12934** [reply]
The creation timestamp is already on over scan.
Or do you mean to include this timestamp also on findings? Like a `scan_created_at` timestamp?

**Reply by ammerzon** [reply]
The second one. To create something like a `scan_created_at` timestamp.

**Reply by J12934** [reply]
Turned this into a feature request #1109 :)
Would be a cool enhancement
